syncer/checkpoint_test.go

```diff
@@ -160,24 +162,25 @@ func (s *testCheckpointSuite) testGlobalCheckPoint(c *C, cp CheckPoint) {
 	s.mock.ExpectQuery(loadCheckPointSQL).WithArgs(cpid).WillReturnRows(sqlmock.NewRows(nil))
 	err = cp.Load(tctx, s.tracker)
 	c.Assert(err, IsNil)
-	cp.SaveGlobalPoint(pos1)
+	cp.SaveGlobalPoint(binlog.Location{Position: pos1})
```

Comment:
We can add some test cases for GTID set later, maybe you can add a TODO in this PR.

Reply:
I add a `TODO` in 35248aa